copybookreader-plugins/src/test/java/co/cask/hydrator/plugin/CopybookTest.java

```diff
@@ -139,13 +130,18 @@ public void testCopybookReaderWithOutputSchema() throws Exception {
     DataSetManager<Table> outputManager = getDataset(outputDatasetName);
     List<StructuredRecord> output = MockSink.readOutput(outputManager);
     StructuredRecord record = output.get(0);
-    Assert.assertEquals("Expected schema", schema, record.getSchema());
+
     Assert.assertEquals("Expected records", 5, output.size());
-    Assert.assertEquals("Expected schema", null, record.get("DATE"));
+    Assert.assertEquals("Output schema", "{\"type\":\"record\",\"name\":\"record\"," +
```

Comment:
should not be checking for schema equality by string comparison. Please construct the actual expected schema and compare schema objects.